src/stampy_nlp/routes.py

```diff
@@ -86,7 +86,7 @@ def literature_api():
     return jsonify(lit_search(query, top_k=top_k))
 
 
-@api.route('/extract', methods=['GET'])
+@api.route('/extract', methods=['GET', 'POST'])
```

Comment:
is POST used?

Reply:
Not right now. From what I gathered from recommended API design, it seems like POST & PUT were meant for updating & creating items, so maybe use them for adding encodings to pinecone?

Reply:
cool. I was wondering whether you wanted to handle long queries, as GETs have a tendency to stop working on some browsers when the URL has more than 4000 characters

Reply:
That's actually what I was thinking when I originally supported POST requests. I took it out when I ran across several guides on good API design. The current endpoint naming convention doesn't really follow the good design guidelines either. The character limit probably isn't as big of an issue for basic Q&A, but may be a problem if we try to support chat conversations and need to keep track of context.